rewatch/src/helpers.rs

```diff
@@ -390,3 +384,10 @@ pub fn get_source_file_from_rescript_file(path: &Path, suffix: &str) -> PathBuf
         &suffix.to_string()[1..],
     )
 }
+
+pub fn is_local_package(workspace_path: &Path, canonical_package_path: &Path) -> bool {
+    canonical_package_path.starts_with(workspace_path)
+        && !canonical_package_path
+            .components()
+            .any(|c| c.as_os_str() == "node_modules")
```

Comment:
For this - I'm not 100% sure if it will work as expected. I think `yarn` links workspaces through `node_modules`. So if I have `packages/A` and `packages/B` which are local, and `B` relies on `A`, then `A` will be inside `/packages/B/node_modules/A` (or `node_modules/A` - not 100% sure) as a symlink.

But - I might be wrong - did you try with `yarn workspaces` by any chance?

Reply:
It's the canonical path, so that shouldn't be in node modules